src/Vizkit3dWorld.cpp

```diff
@@ -18,11 +18,14 @@
 
 namespace vizkit3d_world {
 
-Vizkit3dWorld::Vizkit3dWorld(std::string path, 
+int argc = 1;
+char const*argv[] = { "vizkit3d_world" };
```

Comment:
Why are these globals? I don't see the point but if you really need them to be global at least make they static to avoid collisions